translate/src/core/plural/components/PluralSelector.tsx

```diff
@@ -32,76 +31,57 @@ type InternalProps = Props &
  * Shows a list of available plural forms for the current locale, and allows
  * to change selected plural form.
  */
-export class PluralSelectorBase extends React.Component<InternalProps> {
-  selectPluralForm(pluralForm: number) {
-    if (this.props.pluralForm === pluralForm) {
-      return;
+export function PluralSelectorBase({
```

Comment:
They're effectively only used to provide an easier unit testing surface. I've been able to get rid of a few such wrappers, but not all. It'll be easier once everything is in contexts rather than needing to always create a mock redux state.